jquantsapi/enums.py

```diff
@@ -0,0 +1,19 @@
+from enum import Enum
+
+
+class MARKET_API_SECTIONS(str, Enum):
+    """
+    values of sections for marekt api
```

Comment:
```suggestion
    values of sections for market api
```
@hiohiohio 
めちゃくちゃ細かいですが、誤字っぽかったので直しておきました。